examples/assets/prefab/sphere.ron

```diff
@@ -6,7 +6,7 @@ Prefab (
                 graphics: (
                     mesh: Shape((shape: Sphere(32, 32),)),
                     material: (
-                        albedo: Data(Rgba((0.0, 0.0, 1.0, 1.0,), (),)),
+                        albedo: Data(Rgba((0.0, 0.0, 1.0, 1.0,), (channel: Srgb),)),
```

Comment:
This is unfortunate, but required with our current architecture. We might be infer that from the `Material`.

cc @Rhuagh